graphql-spqr-spring-boot-autoconfigure/src/main/java/io/leangen/graphql/spqr/spring/web/apollo/ApolloMessage.java

```diff
@@ -24,7 +24,9 @@
         @JsonSubTypes.Type(value = InitMessage.class, name = ApolloMessage.GQL_CONNECTION_INIT),
         @JsonSubTypes.Type(value = ApolloMessage.class, name = ApolloMessage.GQL_CONNECTION_TERMINATE),
         @JsonSubTypes.Type(value = ApolloMessage.class, name = ApolloMessage.GQL_STOP),
-        @JsonSubTypes.Type(value = StartMessage.class, name = ApolloMessage.GQL_START)
+        @JsonSubTypes.Type(value = StartMessage.class, name = ApolloMessage.GQL_START),
+        @JsonSubTypes.Type(value = ConnectionErrorMessage.class, name = ApolloMessage.GQL_CONNECTION_ERROR),
```

Comment:
These are intentionally not present in the config, as the client never sends them, so they never need to be deserialized (they're only ever serialized, and this is deserialization config). Do you actually have a case where you need to deserialize these messages?

Reply:
Yeah so I deserialize them in integration tests thats about the only true use case I have as of right now, I just had a test that did "verify that the client got the write message back" from my web socket handler, I don't absolutely need to deserialize its just makes the integration test code easier to read. For now I've just used something like JSONAssert to test that the json payload is what I expected